src/fn/fn-list.scss

```diff
@@ -0,0 +1,97 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-list;
+$fn-list-item-margin: 0.375rem;
+$fn-list-item-focus-margin: 0.125rem;
+
+.#{$block} {
+  @include fn-reset();
+
+  &__item {
+    @include fn-reset();
+    @include fn-flex-vertical-center();
+
+    @include fn-hover() {
+      background: $fn-color-grey-50;
+    }
+
+    @include fn-focus() {
+      border-radius: $fn-list-item-focus-margin;
+      box-shadow: 0 0 0 $fn-list-item-focus-margin $fn-color-blue-500;
+      position: relative;
+    }
+
+    @include fn-selected() {
+      background: $fn-color-blue-100;
+      box-shadow: inset 0 -1px 0 #0070f1;
+    }
+
+    height: 2.75rem;
+    min-width: 20rem;
+    padding: $fn-list-item-margin 0.75rem;
+    background: $fn-color-white;
+    box-shadow: inset 0 -1px 0 $fn-color-grey-100;
+
+    .#{$block}__button {
+      &:not(:last-of-type) {
+        @include fn-set-margin-right($fn-list-item-margin);
+      }
+
+      justify-content: end;
+    }
+
+    .#{$block}__icon {
+      @include fn-set-margin-right(0.625rem);
+      @include fn-set-margin-left(0.25rem);
+
+      color: $fn-color-grey-500;
+    }
+
+    &--has-checkbox {
+      @include fn-set-padding-left(0.625rem);
+
+      .#{$block}__checkbox {
+        @include fn-set-margin-right($fn-list-item-margin);
+      }
```

Comment:
You can avoid this additional class by using:
```
& > div:first-child {
    ....
}
```

This will target the first div that is also a direct child of `fn-list--has-checkbox` and will allow you to get rid of an additional modifier class.